react/components/UI/BorderedLock/index.js

```diff
@@ -1,30 +1,31 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import { translucentGray } from 'react/styles/functions';
-
 import Icon from 'react/components/UI/Icons';
 import Box from 'react/components/UI/Box';
 
 // TODO: All this should be configurable if we need a similar style of icon
 const Lock = styled(Icon).attrs({
   name: 'Lock',
   color: 'gray.medium',
-  size: '10px',
+  size: '0.625rem',
```

Comment:
@broskoski if you need to convert to rem or em depending on the context just divide the number u want by 16.

Reply:
em when you want a value to scale based on the surrounding immediate context (typically padding/margins), rem when you want to scale based on the root font size (which shifts down on mobile)